ntbs-service/Models/NtbsContext.cs

```diff
@@ -36,6 +36,7 @@ public NtbsContext(DbContextOptions<NtbsContext> options)
         public virtual DbSet<Sex> Sex { get; set; }
         public virtual DbSet<PHEC> PHEC { get; set; }
         public virtual DbSet<PostcodeLookup> PostcodeLookup { get; set; }
+        public virtual DbSet<Occupation> Occupation { get; set; }
```

Comment:
Does it need to be public if we're exposing methods to access it below? 
(I completely understand its following the current pattern, but perhaps there's a discrepancy this points to?)

Reply:
Reluctant to change this from public virtual - as I believe it's required for any lazy loading down the line - however as discussed I will in my next changeset create a ReferenceDataRepository, which will mean that viewmodels shouldn't require direct access to the context.